cloudamqp/main.tf

```diff
@@ -0,0 +1,42 @@
+# Create a new cloudamqp instance
+resource "cloudamqp_instance" "instance" {
+  name        = "dev-mqtt"
+  plan        = "squirrel-1"
+  region      = "google-compute-engine::${var.region}"
+  tags        = ["dev"]
+  rmq_version = "3.10.8"
```

Comment:
Are software versions usually specified in `.tf` or `.tfvars` files? I guess we won't find a Dependabot-like for either.

Reply:
dependabot won't work with Terraform code. Each terraform provider or terraform version has to be manually checked. Also in this case changing rmq_version is very risky via automation. Have to test the terraform logic here to see how it works (does it create a new and replace or just update the version)